Sources/AppStoreConnectCLI/Model/PrereleaseVersion.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Foundation
+import SwiftyTextTable
+
+struct PreReleaseVersionDetails: ResultRenderable {
```

Comment:
Can have a discussion if we want to keep the consistency or not, we haven't used keyword `Details` for other models.
```suggestion
struct PreReleaseVersion: ResultRenderable {
```